src/olympia/editors/helpers.py

```diff
@@ -449,15 +449,14 @@ def get_actions(self, request):
                 'minimal': True,
                 'comments': False,
             }
-            # Not implemented yet, will be in #5275.
-            # actions['reject_auto_approved'] = {
-            #     'method': self.handler.reject_auto_approved,
-            #     'label': _('Reject'),
-            #     'details': _('This will reject the specified '
-            #                  'auto-approved versions. The comments will '
-            #                  'be sent to the developer.'),
-            #     'minimal': True,
-            # }
+            actions['reject_multiple_versions'] = {
```

Comment:
Right. It's intended for a full post-review world, to reject one or more versions after they have already been auto-approved (so, typically, the current public ones, and maybe a few older ones).